plugins/webapi/plugin.go

```diff
@@ -75,6 +78,19 @@ func configure(plugin *node.Plugin) {
 		}
 	}
 
+	// Load whitelisted addresses
+	whitelist := append([]string{"127.0.0.1", "::1"}, parameter.NodeConfig.GetStringSlice("api.whitelistedAddresses")...)
+	if len(whitelist) > 0 {
```

Comment:
`len(whitelist)` is always 2 here, so you don't need this check

Reply:
True, I added the two addresses after the condition